mev_inspect/classifiers/specs/cryptopunks.py

```diff
@@ -0,0 +1,21 @@
+from mev_inspect.schemas.traces import (
+    Protocol,
+)
+
+from mev_inspect.schemas.classifiers import (
+    ClassifierSpec,
+    PunkBidClassifier,
+    PunkBidAcceptanceClassifier,
+)
```

Comment:
Classifier should be defined here alongside the spec

`mev_inspect.schemas.classifiers` is just for shared schemas